deltachat-jsonrpc/src/api/types/qr.rs

```diff
@@ -183,6 +198,21 @@ pub enum QrObject {
         /// Authentication code.
         authcode: String,
     },
+    /// Ask the user if they want to revive their own broadcast channel invite QR code.
+    ReviveJoinBroadcast {
+        /// Contact ID.
+        name: String,
+        /// Group ID.
+        grpid: String,
+        /// Contact ID.
+        contact_id: u32,
+        /// Fingerprint of the contact key as scanned from the QR code.
+        fingerprint: String,
+        /// Invite number.
```

Comment:
Btw, is the invite number format documented anywhere? A number which is `String` definitely needs clarification

Reply:
Not sure why it's called number and not code or token; this name comes from the 2018 research project which is now here: https://securejoin.readthedocs.io/.

We could rename 'invite number' throughout the code, but that's not for the PR here.